{{cookiecutter.project_dirname}}/terraform/vars/.tfvars

```diff
@@ -3,6 +3,7 @@
 # django_additional_allowed_hosts=""
 # django_configuration="Remote"
```

Comment:
Non c'entra con la PR ma questa variabile non esiste. Quindi, o la definiamo in `variables.tf` e la utilizziamo in `main.tf` (ma non vedo casi d'uso utili) oppure la rimuoviamo da qua.

Reply:
if we will remove this var from here it could be vanish also from all `variables.tf`